fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/RuleSet.java

```diff
@@ -351,6 +353,11 @@ public List<Rule> getRewriteRulesByType(RuleSetType type) {
         return REWRITE_RULES.get(type);
     }
 
+    public void addMVRules() {
```

Comment:
better to use more specific names. because it may conflict with logical rewrite.

Reply:
fixed